### PR TITLE
Answer

### PR DESCRIPTION
**Pull request by Se-ven**
Initial thoughts

### CODE REVIEW

File: QUESTIONS.md [hunk 1]
```diff
@@ -3,3 +3,20 @@
 What questions can you come up with?
 
 ## Cohort Questions
+
+1. How can we better troubleshoot things in node_modules when it won't load or initialize installed packages?
+
+2. What are some ways I can manage burnout and paralysis analysis?
```
Comment (jessicareynolds) [inline QUESTIONS.md:20]: +1 great question!

File: QUESTIONS.md [hunk 2]
```diff
@@ -3,3 +3,20 @@
...
+
+3. Advice on how to maintain continuity of thought when working on a long-term project?
+
+4. Can we still keep in touch after the cohort ends regarding coding questions/problems?
```
Comment (jessicareynolds) [inline QUESTIONS.md:24]: I know that Hollis suggested we set up a Facebook group for us all to stay connected. I think LinkedIn will be helpful to stay connected with Stack Education.

File: HIGHLIGHTS.md [hunk 3]
```diff
@@ -7,3 +7,20 @@ What highlights can you come up with to share with the class?
 What do you really like? What can you find for fun facts related to any of these topics online?
 
 ## Cohort Highlights
+
+1. [Markdown Guide] (https://www.markdownguide.org/basic-syntax/)
+2. Commit messages written in the imperative
+3. Engineering communication habits, like how to describe what specific problems and what we've already tried, effective problem parsing. 
+4.  ![XKCD, Funny things for everything.] (https://imgs.xkcd.com/comics/real_programmers.png)
+
+5. Playing with the new dog age formula:
```
Comment (jessicareynolds) [inline HIGHLIGHTS.md:19]: Nice! LGTM

File: QUESTIONS.md (hunk 1 above)
Comment (AL743-ni82dy) [inline QUESTIONS.md:20]: I wonder about that always!

File: HIGHLIGHTS.md (hunk 3 above)
Comment (Se-ven) [reply HIGHLIGHTS.md:19]: :)

File: QUESTIONS.md (hunk 1 above)
Comment (hollisjamison) [inline QUESTIONS.md:20]: Best thing to remember is not 1 person is leaving this cohort with 100% knowledge. Everyone will have strengths and weaknesses.

You will never (and no other developer ever does) start a medium+ sized project and finish it in one sitting knowing how to do everything from memory. The important thing is you gain the confidence to find tools to learn things on the fly and use them. Don't be afraid to fail at things, and just keep going until the damn code compiles. 

That being said, if you ever get a hint of burnout, just take a break. The code isn't going anywhere, you can finish it when you get back, and you may even hop back on and realize with a fresh set of eyes that the answer was right in front of you.

**Review by SBucharelli — APPROVED** [review]

**Pull request merged**